### ISSUE TITLE
How to cite this package?

### ISSUE THREAD

**Issue by liupei101** [vegarsti/fhtboost#1]
Hi! vegarsti!

I am writing my paper about survival analysis. And I found your research work about `fhtboost` (the R package and master thesis, maybe...) in this repo.

I wonder how to cite your work about `fhtboost` in my paper? Something like this?
> Lee, M.-L. T., and Whitmore, G. A. 2006. Threshold regression for survival analysis: Modeling event times by astochastic process reaching a boundary. Statistical Science 21(4):501–513.

Look forward to your reply! Thx!

**Comment by vegarsti** [comment]
Hi! Oh my, how did you even find this package? Have you found any use for it? I'm sure it's really hard to figure out how to use it since I haven't documented things, and not sure the structure of the functions make sense yet.

To answer your question, I honestly don't know. I'm sure a proper citation would be if the package was on [CRAN](https://cran.r-project.org). I'm not sure what the convention for citing a package on Github is, but possibly something like this?

```
Stikbakke, Vegard, fhtboost, (2019), GitHub repository, https://github.com/vegarsti/fhtboost
```

(inspired by [this answer](https://academia.stackexchange.com/a/14015) on StackOverflow)

**Comment by liupei101** [comment]
Yes, as you see, I really found it. Such a nice job! ^_^

 Cause I am investigating some existing or latest work about `fht model`. If I don’t understand it wrong, your proposed `fhtboost` improves the traditional `fht model(lee and whitmore 2006)` by using a boosting method to estimate the parameter of the model. And your published R package is also an important contribution. I prepare to add your research work about `fhtboost` into the introduction part in my draft article (titled `"Survival Analysis using A Multi-output Gradient Boosting Decision Tree Method"`).

Thank u so much for your citation example! Maybe you can consider publishing your completed master thesis on [arxiv](https://arxiv.org/). It will make the work known to more researchers, and also be more convenient to cite your thesis.

**Comment by vegarsti** [comment]
Indeed, currently there doesn't exist any method to estimate high-dimensional models, even regularized low-dimensional models, for the First Hitting Time models. So I'm exploring whether multidimensional gradient boosting will work.

I will indeed publish that master's thesis on ArXiV, thanks for the encouragement! I will hand it in by May 15, will let you know!
